app/models/manageiq/providers/amazon/network_manager/refresh_parser.rb

```diff
@@ -350,14 +355,16 @@ def parse_load_balancer_health_check_member(member)
   end
 
   def parse_floating_ip(ip)
-    address = uid = ip.public_ip
+    cloud_network_only = ip.domain["vpc"] ? true : false
```

Comment:
it's a bit weird. Though this code is there from a beginning and works. :-) probably putting a debugger there and check out what is really there will be the best